e2e/utils/touch.js

```diff
@@ -1,10 +1,40 @@
 /* eslint-disable no-console */
 
-const dragCardTo = async(browser, cardSelector, yPos)=>{
-	await browser.executeAsync(async(cardSelectorBrowser, yPosBrowser, done) => {
+const range = function(from, to, step=1) {
+	const ar = [];
+	let localFrom = from;
+
+	ar[0] = localFrom;
+	if (to<localFrom) {
+		while (localFrom-step > to) {
+			ar[ar.length] = localFrom-= step;
+		}
+	} else {
+		while (localFrom+step < to) {
+			ar[ar.length] = localFrom+= step;
+		}
+	}
+	ar[ar.length] = to;
+
+	return ar;
+};
```

Comment:
**[💅🏽  nit-pick]**

I think this kind of utilities is worth to consider an external library doing this for us. I found this lib which seems to have a good adoption. Would its range utility be useful for our use case?
https://mathjs.org/docs/reference/functions/range.html

Reply:
Oh, yeah, now that we have it in our nodejs scope, we can use a library, I did this to be able to execute it inside the browser, but the refactor placed it outside, I'll add the library